config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
+  devise_for :users, :controllers => { registrations: 'registrations' }
+  root 'posts#index'
+  resources :users, only: [:show]
```

Comment:
**REQUIRED**
* This is totally unnecessary since we don't have users controller.
`devise_for :users` is enough in this case.